server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -362,12 +362,7 @@ private void completeFacets(Facets facets, SearchWsRequest request, Request wsRe
     addMandatoryValuesToFacet(facets, LANGUAGES, request.getLanguages());
     addMandatoryValuesToFacet(facets, TAGS, request.getTags());
     addMandatoryValuesToFacet(facets, TYPES, RuleType.names());
-    List<String> actionPlans = Lists.newArrayList("");
-    List<String> actionPlansFromRequest = request.getActionPlans();
-    if (actionPlansFromRequest != null) {
-      actionPlans.addAll(actionPlansFromRequest);
-    }
-    addMandatoryValuesToFacet(facets, DEPRECATED_ACTION_PLANS, actionPlans);
+    addMandatoryValuesToFacet(facets, DEPRECATED_ACTION_PLANS, Collections.<String>emptyList());
```

Comment:
You have well done ! But you should also removed SearchWsRequest#setActionPlans and getActionPlans. 
Moreover I confirm that you can remove this line as no date are returned in this facet => the facet 'actionPlans' is never returned.